auth-web/src/components/auth/common/CardHeader.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
I've already added this component with unit tests and everything a while back with unit tests and everything:
https://github.com/bcgov/sbc-auth/blob/main/auth-web/src/components/CardHeader.vue

Keep in mind that my version has extra functionalities/props like a badge.

Reply:
Thanks. Dunno how I didn't see it.

I'll update this in the follow-on next ticket (bcgov/entity#21815).

Reply:
No problem! Also, I changed the version here to use composition API to stay consistent in AUTH WEB as per a conversation I had with Travis.

Reply:
Yup.
I believe we're going to have a chat on Monday about moving some stuff out of Auth Web and into a new UI project. That new UI will probably be Nuxt. When we port code over, it can be converted to Composition API.